chempy/util/arithmeticdict.py

```diff
@@ -23,11 +23,11 @@ def _itruediv(d1, d2):
 
 
 class ArithmeticDict(defaultdict):
-    """A dictionary which supports arithmetics
+    """A dictionary which supports arithmetic
```

Comment:
Thoughts on this one?  `codespell` is saying that there is no plural form of this word but we can add it to the `--ignore-words-list` if you want to keep it as is.

Reply:
I think your correction is right (I'm no native speaker, but `arithmetic` sounds better to me, and google agrees)